config/environments/development.rb

```diff
@@ -2,6 +2,8 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Aren't we using letter opener?

Reply:
I wanted to test a little bit with the default configuration. I am going to add the gem as well in a moment.

Reply:
Understood. 👍 

Good job.